apps/api/v2/src/modules/slots/inputs/get-available-slots.input.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  IS_ISO8601,
+  IsArray,
+  IsBoolean,
+  IsDateString,
+  IsNumber,
+  IsOptional,
+  IsString,
+} from "class-validator";
+
+export class GetAvailableSlotsInput {
+  @IsDateString()
+  startTime!: string;
+
+  @IsDateString()
+  endTime!: string;
+
+  @IsNumber()
+  @IsOptional()
+  eventTypeId?: number;
+
+  @IsArray({ each: true })
+  @IsString()
+  usernameList?: string[];
+
+  @IsBoolean()
+  @IsOptional()
+  debug?: boolean;
+
+  @IsNumber()
+  @IsOptional()
+  duration?: number;
+
+  @IsBoolean()
+  isTeamEvent!: boolean;
```

Comment:
Curious why the client would have to specify if it’s a team event or not?

Reply:
It feels weird that the person calling it needs to know that versus just passing the eventTypeId and we figure it out

Reply:
That makes sense, I'll change logic for that.